pages/resources/collectibles.mdx

```diff
@@ -0,0 +1,802 @@
+| dark  | ?string | The URL of the dark mode image  |
+
+###### Badge Collectibles Marketing Structure
+
+| Field               | Type    | Description                                                     |
+| ------------------- | ------- | --------------------------------------------------------------- |
+| type                | integer | The [collectibles marketing type](#collectibles-marketing-type) |
+| version             | integer | The version of the badge                                        |
+| dismissible_content | integer | The ID of the dismissible content                               |
+
+###### Banner Collectibles Marketing Structure
+
+| Field              | Type    | Description                                                     |
+| ------------------ | ------- | --------------------------------------------------------------- |
+| type               | integer | The [collectibles marketing type](#collectibles-marketing-type) |
+| version            | integer | The version of the banner                                       |
+| title              | string  | The title of the banner                                         |
+| body               | string  | The body text of the banner                                     |
+| asset              | string  | The URL of the banner image                                     |
+| popout_asset       | string  | The URL of the banner popout image                              |
+| revert_text_color? | boolean | Whether to revert the text color on the banner                  |
+
+###### Coachmark Collectibles Marketing Structure
+
+| Field                 | Type    | Description                                                     |
+| --------------------- | ------- | --------------------------------------------------------------- |
+| type                  | integer | The [collectibles marketing type](#collectibles-marketing-type) |
+| title                 | string  | The title of the coachmark                                      |
+| body                  | string  | The body text of the coachmark                                  |
+| asset_dark            | string  | The URL of the dark mode coachmark image                        |
+| asset_light           | string  | The URL of the light mode coachmark image                       |
+| version               | integer | The version of the coachmark                                    |
+| ref_target_background | object  | The target background configuration                             |
+
+###### Collectible Style Structure
+
+| Field             | Type           | Description                   |
+| ----------------- | -------------- | ----------------------------- |
+| background_colors | array[integer] | An array of background colors |
+| button_colors     | array[integer] | An array of button colors     |
+| confetti_colors   | array[integer] | An array of confetti colors   |
+
+###### Collectible User Discount Structure
```

Comment:
why is this in the marketing object (should probably be made its own object or moved under one of the endpoints)